config/model_configs/box_density_current_test.yml

```diff
@@ -2,7 +2,6 @@ reference_job_id: "les_box"
 initial_condition: "DryDensityCurrentProfile"
 config: "box"
 smagorinsky_lilly: true
-c_smag: 0.25
```

Comment:
Getting `c_smag` from `ClimaParams` is a reasonable change ; do we also need to remove 
https://github.com/CliMA/ClimaAtmos.jl/blob/33a2af5b8ad8398d8b26b9be3f7533cbc09c951d/src/solver/types.jl#L131
from the struct properties ? (Overall the PR LGTM. I think once you make the change to the struct linked above we can merge this)

Reply:
removed this and `Pr_t` too